models/datasets/README.md

```diff
@@ -0,0 +1,150 @@
+# Morpheus Datasets
```

Comment:
In the example workflows themselves (and/or in the model cards), are we also going to link to these datasets?

Reply:
made a new issue for this- https://github.com/nv-morpheus/Morpheus/issues/476